src/content/app/tools/blast/views/blast-submission-results/components/blast-results-per-sequence/BlastResultsPerSequence.tsx

```diff
@@ -37,6 +35,7 @@ import styles from './BlastResultsPerSequence.scss';
 type BlastResultsPerSequenceProps = {
   sequence: {
     id: number;
+    header?: string;
     value: string;
```

Comment:
You now have declared a `SubmittedSequence` type; please use it throughout the PR.

Reply:
i thought i did replace everywhere but looks like i have missed a few places. should all be there now.